### PR TITLE
Fix: update proposal metadata

### PR DESCRIPTION
**Pull request by Chunn**

### CODE REVIEW

**Comment by Chunn** [comment]
@WeiLi512 @parrot9design @BennyOptions I've updated the proposal metadata. However, there are 2 points that need to reconsider:

- StatusMap key is different from status checking. Please check attached images:
evidence1
evidence2

- We've changed "Executed" to "Enacted". It seems we also need to change "Execution Failed" to "Enaction Failed".

Could you help me check? Thank you!

**Comment by parrot9design** [comment]
> We've changed "Executed" to "Enacted". It seems we also need to change "Execution Failed" to "Enaction Failed".

Let’s change it to `Failed to enact` – thanks @Chunn!

**Comment by Chunn** [comment]
>Let’s change it to Failed to enact – thanks @Chunn!
@parrot9design Updated. Please check - thanks

File: src/app/components/ProposalStatusIcon/index.tsx [hunk 1]
```diff
@@ -36,21 +36,35 @@ export function ProposalStatusIcon(props: ProposalStatusProps) {
...
+      const timeLeft = startTime.diff(dayjs().utc(), 'hours');
+      if (timeLeft < 0) return '';
+      return startTime.fromNow(true);
+    }
+    return '';
+  };
 
-  let endTimeStr = platformDay
-    ? dayjs()
+  const endTimeStr = () => {
```
Comment (WeiLi512) [inline src/app/components/ProposalStatusIcon/index.tsx:52]: Define the time formatting functions outside of the ProposalStautsIcon component scope.

**Review by WeiLi512 — CHANGES_REQUESTED** [review]

**Comment by WeiLi512** [comment]
> @WeiLi512 How about StatusMap key? Do we need to update it to be consistent?
> image

I think there isn't `Failed Excuted` or `Failed Excution` status. 
@dbrehmer If a proposal's vote action is not executed because of an [exception](https://github.com/balancednetwork/balanced-contracts/blob/f044f18a94c43c52405b1b77b4134a7d1e710ed0/core_contracts/governance/governance.py#L243
), what status will the proposal have?

**Comment by dbrehmer** [comment]
Currently we have removed the 'Failed Execution' state, but I am thinking that it may be better to handle an error in execution and set the state to 'Failed Execution'. That way it will not be buried as a failed txn that we may not see immediately. If we handle the error and set the status to 'Failed Execution' it will be easier for the frontend to see it and show that status so we know something has to be fixed.

File: src/utils/timeformat.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import dayjs from 'dayjs';
+
+export const formatTimeStr = (targetDay, platformDay, type: 'end' | 'start') => {
```
Comment (WeiLi512) [inline src/utils/timeformat.ts:3]: This function is difficult to understand for me. Could you refactor the code for better understanding?

File: src/utils/timeformat.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  const isStart = type === 'start';
+  const targetTime = dayjs()
+    .utc()
+    .add(targetDay - platformDay - (isStart ? 0 : 1), 'day')
+    .hour(17);
+  const timeLeft = targetTime.diff(dayjs().utc(), 'hours');
+
+  if (timeLeft < 0) return '';
+  if (isStart) return targetTime.fromNow(true);
```
Comment (WeiLi512) [inline src/utils/timeformat.ts:12]: Why does it return in this case? I think we also have to use the format of the end time in this case.
`Starting in 2 days, 2 hours`

**Review by WeiLi512 — CHANGES_REQUESTED** [review]

File: src/utils/timeformat.ts (hunk 3 above)
Comment (WeiLi512) [reply src/utils/timeformat.ts:12]: @parrot9design please confirm the format in this case.

Comment (parrot9design) [reply src/utils/timeformat.ts:12]: It should show the days and hours until voting starts/ends.

**Review by WeiLi512 — APPROVED** [review]

**Review by parrot9design — APPROVED** [review]

**Pull request merged**